src/courtroom.cpp

```diff
@@ -2252,7 +2252,7 @@ void Courtroom::do_effect(QString fx_name, QString fx_sound, QString p_char,
 
 void Courtroom::play_char_sfx(QString sfx_name)
 {
-  sfx_player->play(sfx_name);
+  sfx_player->play(sfx_name, "","",0);
```

Comment:
instead of doing this, you should add default arguments in the function definition of `play()`

Reply:
see https://github.com/AttorneyOnline/AO2-Client/pull/355/commits/59a6295019f7e9c4e2bd85c54678f4767a9e5baa

Reply:
uhhh wait now i understand why you did that... because you want to force the SFX channel to be zero instead of always playing it on the next free channel

Reply:
no, I do want to always play it on the next free channel

Reply:
so what's the point of the `channel` parameter

Reply:
I don't frankly know, most of this code was already here. I'll remove it.

Reply:
see https://github.com/AttorneyOnline/AO2-Client/pull/355/commits/09891684c27aa7e9be2fcf98db144a5bcef05165